pages/column-detail.html

```diff
@@ -0,0 +1,179 @@
+          >幾天一次才剛好
+        </h1>
+        <div class="d-flex flex-column flex-lg-row justify-content-lg-between">
+          <!-- prettier-ignore -->
+          <%- include('./components/breadcrumb', {
+            items: [
+              { name: '植藝生活', link: '#' },
+              { name: '養護專欄', link: '#' },
+              { name: '在陽台種一盆療癒：多肉植物打造你的綠意角落', link: '#' },
+            ],
+          }); -%>
+          <p class="fs-sm fs-lg-8 text-primary mt-6 mt-lg-0">2025/01/01</p>
+        </div>
+      </section>
+      <section class="py-8 py-lg-12">
+        <img
+          class="column-detail-column-7 w-100 object-fit-cover"
+          src="../assets/images/img_column_07.png"
+          alt="column_07"
+        />
+        <h2 class="fs-7 fs-lg-5 noto-serif-tc fw-bold text-primary mt-4 mt-lg-6">
+          多肉植物以「可愛、好養」著稱，<span class="d-block d-lg-none"></span
+          >但最常讓新手困惑的問題，往往就是——到底要多久澆一次水？ 每次要澆多少？
+        </h2>
+        <p class="text-neutral-500 mt-4 pb-3">
+          澆太多會爛根，澆太少又會乾扁，看似簡單的一盆小植物，其實藏著不少學問。這篇文章將帶你了解多肉植物的澆水原則、判斷方法與實用小技巧，讓你的多肉不再枯黃或爆水，永遠保持肉嘟嘟的狀態！
+        </p>
+        <div class="bg-primary-100 mt-4 mt-lg-6 px-4 px-lg-6 py-6">
+          <h2 class="h6 fs-lg-5 text-primary">養多肉的小秘訣</h2>
+          <ul class="column-detail-tips ps-4 mt-4">
+            <li>
+              <p class="fs-lg-7"><span class="fw-bold">盆器排水要好：</span> 底部一定要有洞。</p>
+            </li>
+            <li>
+              <p class="fs-lg-7"><span class="fw-bold">土壤要透氣：</span>建議使用多肉專用介質。</p>
+            </li>
+            <li>
+              <p class="fs-lg-7"><span class="fw-bold">澆水原則：</span> 「乾透再澆透」，不要天天澆。</p>
+            </li>
+            <li>
+              <p class="fs-lg-7"><span class="fw-bold">給光線：</span>陽光足，多肉顏色才會更繽紛。</p>
+            </li>
+            <li>
+              <p class="fs-lg-7"><span class="fw-bold">避免積水與潮濕：</span>是新手最常犯的錯誤！</p>
+            </li>
+          </ul>
+        </div>
+        <div class="row gy-3 gy-lg-0 mt-6 mt-lg-12">
+          <div class="col-lg-6">
+            <img
+              class="w-100 column-detail-img-h200 column-detail-img-lg-h600 object-fit-cover"
+              src="../assets/images/img_column_11.png"
+              alt="column_11"
+            />
+          </div>
+          <div class="col-lg-6">
+            <img
+              class="w-100 column-detail-img-h200 column-detail-img-lg-h600 object-fit-cover"
+              src="../assets/images/img_column_12.png"
+              alt="column_12"
+            />
+          </div>
+        </div>
+        <h2 class="fs-7 fs-lg-5 noto-serif-tc fw-bold text-primary mt-6">先了解多肉的「體質」</h2>
+        <p class="text-neutral-500 mt-4">
+          多肉植物原產於乾旱地區，葉片或莖部能儲存大量水分，因此比一般植物更耐旱但怕積水。<span
+            class="d-block mb-1"
+          ></span>
+          換句話說，它寧可「乾一點」，也不要「濕太久」。多肉的根系喜歡空氣流通、怕悶熱潮濕，一旦長期泡在濕土裡，根部就容易腐爛。
+        </p>
+        <img
+          class="w-100 column-detail-img-h200 column-detail-img-lg-h730 mt-6 mt-lg-12"
```

Comment:
```suggestion
          class="w-100 column-detail-img-h200 column-detail-img-lg-h730 mt-6 mt-lg-12 object-fit-cover"
```
這邊少加了`object-fit-cover`，其他部分沒有問題